canonical-web-and-design/ubuntu.com
# update /robotics illustrations

## Done

Updated illustrations on /robotics according to [brief](https://github.com/canonical-web-and-design/ubuntu.com/issues/6679#issue-566410080)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/robotics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the items in the [issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/6679#issue-566410080) have been properly addressed.


## Issue / Card

Fixes #6679
